girderformindlogger/api/v1/applet.py

```diff
@@ -1641,6 +1644,23 @@ def deactivateApplet(self, applet):
 
         return message
 
+    @access.user(scope=TokenScope.DATA_READ)
+    @autoDescribeRoute(
+        Description('Request id')
+        .param(
+            'request_id',
+            'Request id for applet creation',
+            required=True
+        )
+    )
+    def check_state(self, request_id):
+        cache.create()
+        value = cache.get(request_id)
+        cache.stop()
+        if value is None:
+            return dict()
+        return json.loads(value)
```

Comment:
What do you think about this:
```suggestion
import json
from contextlib import suppress

...

    def check_state(self, request_id):
        cache.create()
        value = cache.get(request_id)
        cache.stop()
        with suppress(TypeError, json.JsonDecodeError):
            return json.loads(value)
        return {}
```